service/resource/vaultcrtv1/metric.go

```diff
@@ -19,7 +19,3 @@ var versionBundleVersionGauge = prometheus.NewGaugeVec(
 	},
 	[]string{"major", "minor", "patch"},
 )
-
-func init() {
-	prometheus.MustRegister(versionBundleVersionGauge)
-}
```

Comment:
There are multiple resources for different versions who emit the same metric. For now I just removed the registration in v1 since this is going away anyway. I am wondering if there is a more graceful way to register metrics multiple times.

Reply:
I reckon we'll probably need to version the metrics :/

Reply:
That would be weird because this means we would lose historical data and have to hustle a lot with keeping dashboards in sync and stuff. Maybe we could wildcard metrics in rge dashboards but I would probably prefer not to. Uncertain.